packages/graphql-generator/src/vendor/@graphql-codegen/core/MAINTENANCE.md

```diff
@@ -0,0 +1,43 @@
+
+This package is being forked for internal use as described in the [README](README.md). This file aims to capture the instructions on
+how to update to any specific version of `@graphql-codegen/core` so that any needed updates can be performed as directly as possible.
+
+### Step 1 - Replace all of the vendor files with the file from the target version of `@graphql-codegen/core`
+- Pick a new target version: `export TARGET_CODEGEN_VERSION=2.6.8`
```

Comment:
I could see getting confused here because our package is also named codegen.

Reply:
Good call